jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -32,7 +33,8 @@
 
 
 @tray_registry('imviz-aper-phot-simple', label="Aperture Photometry")
-class SimpleAperturePhotometry(PluginTemplateMixin, DatasetMultiSelectMixin, TableMixin, PlotMixin):
+class SimpleAperturePhotometry(PluginTemplateMixin, ApertureSubsetSelectMixin,

```

Comment:
I guess technically this PR would block #2666 since we're touching the same kind of code. It makes sense to get this in first.

Reply:
Sure, I don't expect much in terms of conflicts though, do you?

Reply:
Not necessarily, but it does create review churn, as people test our PRs separately and then have to re-test when one is rebased on top of the other one later.